packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -64,59 +108,157 @@ Tests can be executed in any of the following supported form factors:
-  locales: ['en', 'fr'],
-  themes: ['terra-default-theme', 'orion-fusion-theme']
-}
+const { config } = require('@cerner/terra-functional-testing');
 
-Terra.describeTests('Describe tests', testOptions, () => {
-  it('should execute only if form factor is tiny or huge, locale is en or fr, and theme is terra-default-theme or orion-fusion-theme', () => {
+config.serviceOptions = {
+  seleniumVersion: '3.141.59-20210311',
+};
+
+exports.config = config;
+```
+
+## Test Utilities
+
+Test utilities are available to help write tests. These utilities are accessed using the `Terra` global object.
+
+## Describe Helpers
+
+The describe helpers are an alias for the top level `describe` block used within spec files. The describe helpers provide useful features for limiting a set of tests to specific form factors, locales, and themes.
+
+### Terra.describeViewports
+
+<Notice variant="important" ariaLevel="3">
+  We recommended using Terra.describeTests instead of Terra.describeViewports. The same features in Terra.describeViewports can be achieved with Terra.describeTests.
+</Notice>
+
+The `Terra.describeViewports` utility defines a list of form factors the tests are enabled for. By default, all tests will be run against each provided viewport. If a form factor is specified via the cli the describe viewpoint helper works as a filter and only allows the tests to execute if they match the current form factor.
+
+```js
+/**
+ * Executes all tests for each defined viewport.
+ * @param {string} title - The describe block title.
+ * @param {string[]} viewports - A list of viewports. [tiny, small, medium, large, huge, enormous]
+ * @param {function} fn - The block of tests to execute against each viewport.
+ */
+Terra.describeViewports(title, [viewports], fn);
+```
+
+<Notice variant="caution" ariaLevel="3">
+  Describe helpers should not be nested.
+</Notice>
+
+```js
+// The following example will scope the tests to run only for the `tiny` and `huge` form factors. The tests will not run for any other form factor.
+
+Terra.describeViewports('Describe viewports title', ['tiny', 'huge'], () => {
+  it('should test tiny and huge screens', () => {
     browser.url('/testing/route/');
-    Terra.validates.element('describe tests');
+
+    Terra.validates.screenshot('describe viewports');
+  });
+});
+
+// Multiple describe helpers can be used sequentially in the same spec file as long as they are not nested.
+
+Terra.describeViewports('Describe viewports title', ['small'], () => {
+  it('should test small screens', () => {
+    browser.url('/testing/route/small/');
+
+    Terra.validates.screenshot('describe viewports');
   });
 });
 ```
 
-The below describe block is not qualified to run because at least one of the form factors, locales, or themes provided to `describeTests` does not match the CLI options specified in the test runner.
-```sh
-terra wdio --formFactors small --locales en --themes terra-default-theme
+### Terra.describeTests
```

Comment:
We should probably add the 'don't nest' notice for terra.describeTests too.

Reply:
Added a caution to not nest:

https://github.com/cerner/terra-toolkit/pull/610/commits/36606851682a8a89e5cc76e7a6ec39436bd45d0c